hooks/useStore.js

```diff
@@ -1,20 +1,42 @@
 import create from 'zustand';
 import { nanoid } from 'nanoid';
+import { persist } from 'zustand/middleware';
 
-const useStore = create(set => {
-  return {
-    recipeList: [
-      { id: nanoid(), recipeTitle: 'Cheesecake' },
-      { id: nanoid(), recipeTitle: 'Carrotcake' },
-    ],
-    addRecipe: recipeTitle => {
-      set(state => {
-        return {
-          recipeList: [{ id: nanoid(), recipeTitle }, ...state.recipeList],
-        };
-      });
+const useStore = create(
+  persist(
+    set => {
+      return {
+        recipeList: [
+          { id: nanoid(), recipeTitle: 'Cheesecake' },
+          { id: nanoid(), recipeTitle: 'Carrotcake' },
+        ],
+        addRecipe: recipeTitle => {
+          set(state => {
+            return {
+              recipeList: [{ id: nanoid(), recipeTitle }, ...state.recipeList],
+            };
+          });
+        },
+      };
     },
-  };
-});
+    {
+      name: 'recipeList',
```

Comment:
code provided by Greg in order to prevent hydration issue

Reply:
looks good 🪄